jwk/handler_test.go

```diff
@@ -50,7 +50,7 @@ func TestHandlerWellKnown(t *testing.T) {
 		err = json.NewDecoder(res.Body).Decode(&known)
 		require.NoError(t, err, "problem in decoding response")
 
-		require.Len(t, known.Keys, 1)
+		require.GreaterOrEqual(t, len(known.Keys), 1)
```

Comment:
There are now both the OIDC and JWT keys present, which is why we need to check `>=` here.

Reply:
Thank you, very helpful